publish.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+##############################################################################################
```

Comment:
Yes, I will add section to the README, modelled on this: https://github.com/aws-samples/amazon-transcribe-post-call-analytics/blob/develop/README.md#optional-build-and-publish-pca-cloudformation-artifacts 

> My understanding is that the script should ideally be run by "automation" after a push to main in order to publish the CF template. What do you think?

I don't think so, because we need to publish it to public buckets using temp credentials... Also don't want it publishing after minor commits like README improvements.   I think we just run it manually, for now, as part of our release process.
Also, customers can run it to publish the artifacts to their own S3 buckets, if they create forks with customizations.